LICENSE

```diff
@@ -1,19 +1,19 @@
-Cppyright (c) 2015, The GAP Group
+Copyright (c) 2015, The GAP Group
```

Comment:
Perhaps this line should also be changed? 2015 seems wrong, perhaps 2017-2018 -- and "Markus Pfeiffer" instead of "The GAP Group"?